index.html

```diff
@@ -75,5 +75,251 @@ <h4>LETS CONNECT</h4>
+                    </div>
+                    <div class="mini-description">
+                      <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                      </p>
+                    </div>
+                    <div class="language-tags">
+                      <ul>
+                        <div>
+                          <li>html</li>
+                        </div>
+                        <div>
+                          <li>css</li>
+                        </div>
+                        <div>
+                          <li>javaScript</li>
+                        </div>
+                      </ul>
+                    </div>
+                    <div class="project-detail-btn">
+                      <button>See Project</button>
+                    </div>
+                  </div>
+                </div>
+             </div>
+             <div class="project-item">
+                <div class="project-image">
+                  <img src="./assets/fouth-image.png" alt="fourth project image"/>
+                </div>
+                <div>
+                  <div class="project-details">
+                    <div>
+                      <h3>Multi-Post <br/> Stories</h3>
+                    </div>
+                    <div class="canopy-para">
+                      <p><span>CANOPY</span> <img src="./assets/Counter.png" alt="a picture dot"> Back end dev <img src="./assets/Counter.png" alt="a picture dot"> 2015</p>
+                    </div>
+                    <div class="mini-description">
+                      <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                      </p>
+                    </div>
+                    <div class="language-tags">
+                      <ul>
+                        <div>
+                          <li>html</li>
+                        </div>
+                        <div>
+                          <li>css</li>
+                        </div>
+                        <div>
+                          <li>javaScript</li>
+                        </div>
+                      </ul>
+                    </div>
+                    <div class="project-detail-btn">
+                      <button>See Project</button>
+                    </div>
+                  </div>
+                </div>
+             </div>
+       </section>
+      <section class="about-myself-section">
+        <div class="about-myself-description">
+          <div>
+            <h2>About <br/> Myself</h2>
+          </div>
+          <div>
+            <p>
+               Hello i’m a software developer! I can help you build a product , feature or website Look through some of my work and
+              experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
```

Comment:
- [ ] You generally designed this part very well, but to improve it you can follow the Figma design spacing. Please take a look at the below screenshot.

![image](https://user-images.githubusercontent.com/34779928/192034902-a9c3a7a9-4cfe-4fdd-a6a8-b2183c188916.png)